### DISCUSSION TITLE
Question: How to close a ticket without tracking time

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#14252] [How to] [answered]
Is there a way to configure the system so that I can close a ticket without having to enter a time tracker entry?

**Comment by rskrzypczak** [comment, marked as answer]
Yes.
`config/Modules/HelpDesk.php`

> /**` When closing the ticket, check if has time control */
public static $CHECK_IF_RECORDS_HAS_TIME_CONTROL = false;
